TEST_SETUP.md

```diff
@@ -30,8 +30,12 @@
    | `OPERATOR_KEY_A`    | This is the `DER Encoded Private Key` which can be found in your account portal. |
    | `HEX_PRIVATE_KEY_A` | This is the `HEX Encoded Private Key` which can be found in your account portal. |
    | `HEX_PRIVATE_KEY_B` | This is the `HEX Encoded Private Key` which can be found in your account portal. |
+   | `HEX_PRIVATE_KEY_C` | This is the `HEX Encoded Private Key` which can be found in your account portal. This key is used in the voting example. |
```

Comment:
This isn't sustainable.
At this point this env variable should be an array.
Can you explore using a `HEX_PRIVATE_KEYS` that holds a list of accounts

Reply:
Done.